addons/l10n_ar_withholding/models/l10n_ar_earnings_scale.py

```diff
@@ -0,0 +1,47 @@
+from odoo import models, fields
+
+
+class L10nArEarningsScale(models.Model):
+    _name = 'l10n_ar.earnings.scale'
+    _description = 'l10n_ar.earnings.scale'
+
+    name = fields.Char(required=True)
+    line_ids = fields.One2many('l10n_ar.earnings.scale.detail', 'scale_id')
+
+
+class L10nArEarningsScaleDetail(models.Model):
+    _name = 'l10n_ar.earnings.scale.detail'
```

Comment:
```suggestion
    _name = 'l10n_ar.earnings.scale.line'
```
What other details do we have?